### PR TITLE
Fix a bug

### PR DESCRIPTION
**Pull request by Sizheng-Ma**
Pointed out by Max Isi:

> The sampling frequency should be 2*fNyq, i.e., 2*fMax, it shouldn't depend on what you call f_min. Any well-formed frequency series has to start at f = 0, for it to inverse-FFT into a well-formed time series. If your PSD frequency-series starts at some nonzero value, that just means we are assuming all values of the PSD below that fmin to be infinity / zero, but the corresponding sampling frequency would still be 2*fmax, not 2*(fmax - fmin).

The first two commits fix this bug and add more checks for the psd/asd file.

The third commit adds two more parameters: `minimum_frequency` and `maximum_frequency` to `set_bilby_ifo`.

### CODE REVIEW

**Comment by Sizheng-Ma** [comment]
@Lu-Neil Please take a look to see if you agree with these changes.

**Comment by maxisi** [comment]
@Sizheng-Ma @Lu-Neil please merge this ASAP so we can use the hash as reference for the review

File: qnm_filter/bilby_helper.py [hunk 1]
```diff
@@ -82,6 +82,8 @@ def set_bilby_ifo(
     sampling_frequency,
     duration,
     start_time,
+    minimum_frequency,
```
Comment (Lu-Neil) [inline qnm_filter/bilby_helper.py:85]: is it worth having default values?

**Review by Lu-Neil — APPROVED** [review]

File: qnm_filter/bilby_helper.py (hunk 1 above)
Comment (Sizheng-Ma) [reply qnm_filter/bilby_helper.py:85]: I'd rather not. The desired value may vary from case to case. The current way can remind ourselves each time it's used.

**Comment by Sizheng-Ma** [comment]
See [this notebook](https://git.ligo.org/cbc-testinggr/reviews/qnm-rational-filter/-/wikis/Validating-Noise-infrastructure-of-qnm_filter) for checks.

**Pull request merged**